manifests/stub.pp

```diff
@@ -32,6 +32,7 @@
   Variant[Array[Unbound::Address], Unbound::Address] $address,
   # lint:ignore:quoted_booleans
   Variant[Boolean, Enum['true', 'false']]            $insecure    = false,
+  Variant[Boolean, Enum['true', 'false']]            $no_cache    = false,
```

Comment:
Personally i would just make this Boolean, i think there was a need to make the other one a variant so as to make it backwards compatible but as this is a new param there is no need for that

Reply:
i'd do that in future release, where we drop this type of backwards compatibility

Reply:
I'll keep the consistency here with the other param, but I agree that deprecating this to favor the boolean seems good longer term.

Reply:
the next release will be a major one, so now might be the time to make it correct for the new param and fix the others in a separate PR?